tensorflow/docs_src/install/install_linux.md

```diff
@@ -524,10 +524,12 @@ on your system:
   use the following command for CUDA Toolkit >= 8.0:
 
 <pre class="prettyprint lang-bsh">
-  <code class="devsite-terminal">sudo apt-get install cuda-command-line-tools</code>
+  <code class="devsite-terminal">sudo apt-get install cuda-command-line-tools-8-0</code>
 </pre>
 
-Add this path to the `LD_LIBRARY_PATH` environmental variable:
+for Cuda 8.0, or cuda-command-line-tools-9-0 or 9-1 as appropriate for your cuda version.
```

Comment:
Is there official doc on this?

Reply:
Many (like me) who hit this issue 
[16214](https://github.com/tensorflow/tensorflow/issues/16214)
successfully resolve by using cuda-command-line-tools-9-0  or the like instead of cuda-command-line-tools

Reply:
The trick is to install cuda / toolkit with deb (network) and not with run (local).